### DISCUSSION TITLE
Scammers using dub.co

### DISCUSSION THREAD

**Discussion by harshsbhat** [dubinc/dub#1285] [Q&A] [answered]
I just got a message from some scammer ( attached ss) 

![Screenshot_2024-09-24-13-30-51-64_b783bf344239542886fee7b48fa4b892.jpg](https://github.com/user-attachments/assets/bfc903e6-be36-4fa9-92ff-6f2d7bd3ad4c)

he is using dub to hide his shady looking link

can we make a form where users can report links like these? which after a certain number of reports will be banned and after a certain number of banned links we can ban the accounts maybe

**Comment by harshsbhat** [comment, marked as answer]
This form is the answer:

https://dub.co/legal/abuse

@steven-tey  can you mark this as the answer to my question? So that future users can know about this feature

**Reply by steven-tey** [reply]
Done, thank you!
